### ISSUE TITLE
Configuring external access with load balancers

### ISSUE THREAD

**Issue by sweetkane** [bitnami/charts#22534]
### Name and Version

bitnami/kafka:3.6.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Cluster is hosted on Azure AKS

### Are you using any custom parameters or values?

2. Technically I'm using terraform, but currently the helm command to apply thse chart would look like this:
```
helm install kafka bitnami/kafka \
    --set controller.replicaCount=1 \
    --set externalAccess.enabled=true \
    --set externalAccess.controller.service.type=LoadBalancer \
    --set externalAccess.broker.service.type=LoadBalancer \
    --set-json='externalAccess.broker.service.loadBalancerNames=["EXTERNAL"]'
```

### What is the expected behavior?

What I'm going for is the most basic possible POC of a kafka server on Azure which can be accessed from my local machine.

### What do you see instead?

Currently I am able to connect the cluster and do arbitrary things like checking the version, but I can't send new messages. I assume this is because of my advertised listeners configuration giving my external client the wrong broker address. My problem is I cannot seem to figure out how to correctly set the advertised listeners so that my external client can connect to the right brokers. 

An added wrinkle is that the broker's external IP is set by Azure when installing the chart, so it is not available until after the installation.

### Additional information

I don't think this is a complex or niche use case by any means, so I'm guessing (hoping) the answer will be straightforward.

TLDR:
- Setup: basic kafka installation on AKS
- Problem: I can connect to server, but sending a message hangs
- My Questions:
  - is this due to the advertised listeners configuration?
  - if so, how can I setup my advertised listeners to solve the problem? Especially when I don't initially know the external IP.
  - if not, what am I doing wrong and how can I get this simple POC working?

Thank you for your time.

**Comment by jotamartos** [comment]
Hi @sweetkane,

Sorry for the delay in getting back to you. There is a similar case in our repo and our team already helped this other user, can you take a look at it? 

https://github.com/bitnami/charts/issues/22003

I also suggest you check if you get the same error in other k8s clusters (minikube or other cloud provider) to confirm if it's a problem with the application or a networking issue with the cluster you are using. 

Thanks
